frontend/fingertips-frontend/components/pages/chart/chart.test.tsx

```diff
@@ -0,0 +1,39 @@
+import { render, screen } from '@testing-library/react';
+import { Chart } from '@/components/pages/chart/index';
+import { expect } from '@jest/globals';
+import { WeatherForecast } from '@/generated-sources/api-client';
+
+const mockData: WeatherForecast[] = [
+  {
+    date: new Date('2024-11-01T00:00:00.000Z'),
+    temperatureC: -30,
+    temperatureF: -21,
+    summary: 'Freezing',
+  },
+  {
+    date: new Date('2024-11-01T00:00:00.000Z'),
+    temperatureC: 0,
+    temperatureF: 32,
+    summary: 'Bracing',
+  },
+];
+
+test('should render the Highcharts react component', () => {
+  render(<Chart data={mockData} />);
+  const highcharts = screen.getByTestId('highcharts-react-component');
+  expect(highcharts).toBeInTheDocument();
+});
```

Comment:
I think this could reference the `lineChart-component` data-test-id value, so we don't have to specifically check for Highcharts presence at this level (we could porentially change the implementation of the chart to use another library, and we wouldn't want the chart page test to fail).

```suggestion
test('should render the line chart', () => {
  render(<Chart data={mockData} />);
  const lineChart = screen.getByTestId('lineChart-component');
  expect(lineChart).toBeInTheDocument();
});
```